workloads/hyper-scale/run.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+source ./common.sh
+
+if [ $# -eq 0 ]; then
+    echo "Missing Argument"
+    echo "Run './run.sh build' to install hypershift and hosted clusters"
+    echo "Run './run.sh clean' to cleanup already installed resources"
+    exit 1
+fi
+
+prep
+if [[ "$1" == "clean" ]]; then
+    printf "Running Cleanup Steps"
+    setup
+    cleanup
+elif [[ "$1" == "build" ]]; then
+    setup
+    install
+
+    export NODEPOOL_SIZE=$COMPUTE_WORKERS_NUMBER
```

Comment:
We should probably ensure that the worker count is divisible by 3 if HA is enabled

Reply:
On Hypershift model, on HA `HostedControlPlane` pods are create in x3 as pods on the management cluster, so workers need not to be that way. I believe once @smalleni had a hostedcluster with 1 or 2 workers, isn't it ?

Reply:
I also confirmed with engineers https://coreos.slack.com/archives/C01C8502FMM/p1649344956868999, `HostedCluster` worker nodes need not to be like that. At least not yet :smile: